ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2

```diff
@@ -401,16 +401,17 @@ host_monitoring_cron:
 - name: check if zabbix monitored all the node
   hour: "*/6"
   job: ops-runner -f -s 15 -n cszic.openshift.zabbix.inventory.info /usr/bin/cron-send-zabbix-inventory-check --clusterid {{ osohm_cluster_id }}
+{% if osohm_cluster_id == 'reg-aws' %}
```

Comment:
why not 
```
{% if osohm_cluster_id != 'reg-aws' %}
.
.
.
{% endif %}
```

Reply:
yeah, interesting logic, what the benefit than `!=`?

Reply:
I am not sure its != or <>

Reply:
https://github.com/openshift/openshift-tools/blob/a9694642353823d0a0e53cc535c86c986d08a586/ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2#L72